pkg/cgroup/linux_test.go

```diff
@@ -0,0 +1,94 @@
+// +build linux
+
+/*
+ * Minio Cloud Storage, (C) 2017 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cgroup
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+// Testing proc cgroup.
+func TestProcCGroup(t *testing.T) {
+	tmpPath, err := ioutil.TempFile("", "cgroup")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.Remove(tmpPath.Name())
+
+	cgroup := `
+11:memory:/user.slice
+10:blkio:/user.slice
+9:hugetlb:/
+8:net_cls,net_prio:/
+7:perf_event:/
+6:pids:/user.slice/user-1000.slice
+5:devices:/user.slice
+4:cpuset:/
+3:cpu,cpuacct:/user.slice
+2:freezer:/
+1:name=systemd:/user.slice/user-1000.slice/session-1.scope
+`
```

Comment:
Same process can have different controllers in different groups as well. But ending path is always unique